app/services/journeyAnswers/ExpensesAnswersService.scala

```diff
@@ -17,25 +17,54 @@
 package services.journeyAnswers
 
 import cats.data.EitherT
-import models.common.{BusinessId, Mtditid, TaxYear}
+import cats.implicits._
+import connectors.SelfEmploymentBusinessConnector
+import models.common.JourneyName.{ExpensesTailoring, GoodsToSellOrUse}
+import models.common.TaxYear.{endDate, startDate}
+import models.common._
+import models.connector.api_1894.request.{CreateSEPeriodSummaryRequestBody, CreateSEPeriodSummaryRequestData, FinancialsType}
 import models.domain.ApiResultT
 import models.error.ServiceError
-import models.frontend.expenses.{ExpensesTailoringAnswers, GoodsToSellOrUseJourneyAnswers}
+import models.frontend.expenses.ExpensesTailoringAnswers
+import models.frontend.expenses.goodsToSellOrUse.GoodsToSellOrUseJourneyAnswers
+import play.api.libs.json.Json
+import repositories.JourneyAnswersRepository
+import uk.gov.hmrc.http.HeaderCarrier
 
 import javax.inject.{Inject, Singleton}
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
 
 trait ExpensesAnswersService {
   def saveAnswers(businessId: BusinessId, taxYear: TaxYear, mtditid: Mtditid, answers: ExpensesTailoringAnswers): ApiResultT[Unit]
-  def saveAnswers(businessId: BusinessId, taxYear: TaxYear, mtditid: Mtditid, answers: GoodsToSellOrUseJourneyAnswers): ApiResultT[Unit]
+  def saveAnswers(businessId: BusinessId, taxYear: TaxYear, mtditid: Mtditid, nino: Nino, answers: GoodsToSellOrUseJourneyAnswers)(implicit
+      hc: HeaderCarrier): ApiResultT[Unit]
 }
 
 @Singleton
-class ExpensesAnswersServiceImpl @Inject() (implicit ec: ExecutionContext) extends ExpensesAnswersService {
+class ExpensesAnswersServiceImpl @Inject() (
+    businessConnector: SelfEmploymentBusinessConnector,
+    repository: JourneyAnswersRepository
+)(implicit ec: ExecutionContext)
+    extends ExpensesAnswersService {
   def saveAnswers(businessId: BusinessId, taxYear: TaxYear, mtditid: Mtditid, answers: ExpensesTailoringAnswers): ApiResultT[Unit] =
-    EitherT.right[ServiceError](Future.successful(()))
+    EitherT
+      .right[ServiceError](
+        repository.upsertData(mtditid, taxYear, businessId, ExpensesTailoring, Json.toJson(answers))
+      )
+      .void
 
-  def saveAnswers(businessId: BusinessId, taxYear: TaxYear, mtditid: Mtditid, answers: GoodsToSellOrUseJourneyAnswers): ApiResultT[Unit] =
-    EitherT.right[ServiceError](Future.successful(()))
+  def saveAnswers(businessId: BusinessId, taxYear: TaxYear, mtditid: Mtditid, nino: Nino, answers: GoodsToSellOrUseJourneyAnswers)(implicit
+      hc: HeaderCarrier): ApiResultT[Unit] = {
+    val financials  = Json.toJson(answers).as[FinancialsType]
```

Comment:
I think the reason why it is currently working end to end with the stub is because the request schema to 1894 has all optional fields (except for from/to dates), that's why it doesn;t error on` None` for `financials.`